doc/reference/interactive_console.rst

```diff
@@ -120,3 +125,27 @@ call was ``console.set_default_output('lua')``.
                 | :code:`...`
 
             -   :code:`\x81 \xa3 \x6b \x65 \x79 \x01`
+
+.. _interactive_console-shortcuts:
+
+Keyboard shortcuts
+------------------
+
+..  list-table::
+    :widths: 25 75
+    :header-rows: 1
+
+    *   - Keyboard shortcut
+        - Effect
+
+    *   - ``CTRL+C``
+        - Discard current input with the ``SIGINT`` signal in the console mode and
+          jump to a new line with a default prompt.
+
+    *   - ``CTRL+D``
+        - Quit Tarantool interactive console.
+
+..  important::
+
+    Keep in mind that ``CTRL+C`` shortcut will shut Tarantool down if there is any currently running command
+    in the console.
```

Comment:
I would add that SIGINT shuts the instance down if it is running in a daemon mode.

Reply:
Added. Thanks for the review!